core-primitives/stf-interface/src/lib.rs

```diff
@@ -145,3 +165,22 @@ impl ShardCreationInfo {
 		}
 	}
 }
+
+pub fn parentchain_mirror_prefix(parentchain_id: &ParentchainId) -> String {
+	format!("L1MirrorFor{:?}", *parentchain_id)
+}
```

Comment:
in earlier PoCs, we mirrored keys 1:1 but that's pretty dangerous as people might instantiate identical pallets on L2. A unambiguous prefix should fix this

Reply:
I guess it was initially the idea for encointer to get the L1 data and put it into the identical pallet of L2. But I do agree that this is more dangerous than useful in general.

Reply:
yes, but even back then this was probably a bad idea because of the risk of storage migrations. Now we can test decoding specifically in  a controlled manner